components/TaskCard/index.js

```diff
@@ -1,17 +1,47 @@
-import { useState } from "react";
+import { useEffect, useState } from "react";
 
 import { Tasks, Task, TaskListItem, Label, Labels } from "./TaskCard.styled";
 
 export const TaskCard = ({ project, index }) => {
   const [visible, setVisible] = useState(false);
+  const [checked, setChecked] = useState(false);
+  const [projectsData, setProjectsData] = useState([]);
+
+  useEffect(() => {
+    const storedProjects = localStorage.getItem("projectsData");
+    if (storedProjects) {
+      setProjectsData(JSON.parse(storedProjects));
+    } else {
+      setProjectsData([]);
+    }
+  }, []);
+
+  console.log(project, index);
+  const { projectTitle, tasks, projectDue, projectSize, status, id } = project;
+
+  useEffect(() => {
+    setChecked(status);
+  }, [status]);
+
   const projectClickHandler = () => {
     setVisible(!visible);
   };
-  console.log(project, index);
-  const { projectTitle, tasks, projectDue, projectSize } = project;
+
+  const onChangeCheckboxHandler = async () => {

```

Comment:
`async` is not necessary here. You're not doing any asynchronous operations that would need an `await`.

Reply:
thanks I got it